bears/general/LineCountBear.py

```diff
@@ -17,16 +17,37 @@ def _get_blank_line_count(self, file):
             list(filter(lambda x: re.match(r'^\s*$', x), file)))
         return num_blank_lines
 
+    def _check_correct_range(self, min_lines_per_file, max_lines_per_file):
+        if min_lines_per_file > max_lines_per_file:
+            print(min_lines_per_file, max_lines_per_file)
+            raise ValueError('Allowed maximum lines per file({}) is smaller '
+                             'than minimum lines per file({}) and this is '
+                             'prohibited'.format(max_lines_per_file,
+                                                 min_lines_per_file))
+        return
```

Comment:
Tbh, I wouldn't write a special function for that this is a simple "if condition -> raise"